javascript/ql/src/Statements/UselessRangeCheck.qhelp

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+<overview>
+<p>
+If a condition always evaluates to true or always evaluates to false, this often indicates
```

Comment:
Suggestion: adding a comma here would clarify things (but this is just a suggestion).
this often indicates incomplete code or a latent bug, and should be examined carefully.